### ISSUE TITLE
Update ReadMe

### ISSUE THREAD

**Issue by hamed-rv** [rvhamed/BarcodeX#4]
**_ReadMe.md_** file should be refactored. Descriptions and abilities for the project should be added in a pretty way.

**Comment by rioX432** [comment]
I think below text is description.
Should I add some text here ?

![image](https://user-images.githubusercontent.com/34413373/105112000-702ea000-5b05-11eb-94dd-e55889e222b3.png)

And what does abilities mean ? What should I write ?

**Comment by hamed-rv** [comment]
@omooooori Yes, this is a short description, but you can make it better. Also, you can add a section for implementation. How users can implement it in their own projects. It has other abilities such as Troch, etc.
